test/node/ssr.test.tsx

```diff
@@ -23,14 +22,10 @@ function App() {
   return <DataGrid columns={columns} rows={rows} />;
 }
 
-// biome-ignore lint/suspicious/noSkippedTests: <explanation>
-test.skip('basic server-side rendering (SSR) support', () => {
+test('basic server-side rendering (SSR) support', () => {
   // make sure we're not running in the JSDOM environment
```

Comment:
```suggestion
```
We do not have JSDOM package anymore. Can we remove this comment?

Reply:
Tweaked it